scout/advanced-image-analysis.md

```diff
@@ -0,0 +1,113 @@
+---
+description: Image analysis with Docker Scout
+keywords: scanning, vulnerabilities, Hub, supply chain, security
+title: Advanced image analysis
+---
+
+> **Note**
+>
+> Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)
+> product, and requires a Pro, Team, or Business subscription.
+
+Advanced image analysis is a Docker Scout feature for Docker Hub.
+
+When you activate Advanced image analysis for a repository, new tags are
+automatically analyzed when you push to that repository. Information about
```

Comment:
Can we add something like "... and analysis will continue to highlight when new CVEs affect the image without the need to re-push", just to highlight the evergreen nature of the image analysis